### PR TITLE
Create 2018-09-24--go-core-dev-team-weekly.md

### PR DESCRIPTION
**Pull request by eingenito**
Please leave your notes if you haven't had a chance to yet. Thanks!

### CODE REVIEW

File: meeting-notes/2018/2018-09-24--go-core-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+- **Lead:** Erik Ingenito
+- **Notetaker:** Molly (thanks!)
+- **Attendees:**
+  - _attendee names..._
+  - @Magik6k
+  - @stebalien
+  - @momack2
+  - @michaelavila
+  - @warpfork
```
Comment (kevina) [inline meeting-notes/2018/2018-09-24--go-core-dev-team-weekly.md]: Please add: @kevina

**Comment by schomatis** [comment]
@eingenito I pushed my notes to the [`create-2018-09-24-go-core-notes`](https://github.com/ipfs/pm/tree/create-2018-09-24-go-core-notes) branch but I don't see them reflected in this PR, do you know what step I'm missing here?

**Comment by diasdavid** [comment]
@schomatis post your notes here as a comment so that we can add them. Not sure what happened there, maybe you pushed to a fork?

**Comment by schomatis** [comment]
https://github.com/ipfs/pm/commit/ab2bde61b204fbd407d12cfb14f3fadbf8c5c18a

> maybe you pushed to a fork?

Seems to be the main `ipfs/pm` repo. (Is the "Allow edits from maintainers" option checked?)

File: meeting-notes/2018/2018-09-24--go-core-dev-team-weekly.md (hunk 1 above)
Comment (eingenito) [reply meeting-notes/2018/2018-09-24--go-core-dev-team-weekly.md]: Will do

**Comment by eingenito** [comment]
Thanks @schomatis. I'll merge your updates. Not why you couldn't write to this branch.

**Pull request merged**